modules/mutect2/2.0/mutect2.smk

```diff
@@ -212,7 +230,7 @@ rule _mutect2_run_no_normal:
         gatk Mutect2 --java-options "-Xmx{params.mem_mb}m" 
         {params.opts} -I {input.tumour_bam} -R {input.fasta} 
         -O {output.vcf} --germline-resource {input.gnomad} 
-        -L {wildcards.chrom} {params.interval_arg}
+        -L {wildcards.chrom} {params.interval_arg} -L {input.capture_arg}
         -pon {input.pon} --f1r2-tar-gz {output.f1r2}
         > {log.stdout} 2> {log.stderr}
         """)
```

Comment:
Actually it's probably the functions below `def _mutect2_get_chr_vcfs(wildcards)` and `def _mutect2_get_chr_tbis(wildcards)` that need to be updated to subset the chromosome list while considering capspace and candidate positions

Reply:
I ended up keeping these functions as is - because they read in the file with chromosomes produced by checkpoint. Instead, I modified the checkpoint to address this. This is resolved now